subscription-service/src/test/resources/ngsild/subscription_with_conflit_attribute.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
try to find a better name for this file. one that indicates why it is not a correct subscription.

Reply:
Maybe 'subscription_with_invalid_attribut' ?

Reply:
still too vague, we can have many cases of invalid subscriptions... `subscription_with_conflicting_timeInterval_watchedAttributes`?

Reply:
I valid